tests/MongoDB.Driver.Core.Tests/Core/Operations/EndTransactionOperationTests.cs

```diff
@@ -149,8 +184,9 @@ public class AbortTransactionOperationTests
         public void constructor_should_initialize_instance()
         {
             var writeConcern = new WriteConcern();
+            var recoveryToken = BsonDocument.Parse("{ generalOrder: 1 }");
```

Comment:
Avoid parsing for simple cases:

    var recoveryToken = new BsonDocument("generalOrder", 1);

Reply:
Fixed